Gemfile.lock

```diff
@@ -149,45 +175,47 @@ GEM
       sprockets (>= 3.0.0)
     sqlite3 (1.4.2)
     thor (1.1.0)
-    thread_safe (0.3.6)
     tilt (2.0.10)
     turbolinks (5.2.1)
       turbolinks-source (~> 5.2)
     turbolinks-source (5.2.0)
-    tzinfo (1.2.9)
-      thread_safe (~> 0.1)
+    tzinfo (2.0.4)
+      concurrent-ruby (~> 1.0)
     uglifier (4.2.0)
       execjs (>= 0.3.0, < 3)
-    web-console (3.7.0)
-      actionview (>= 5.0)
-      activemodel (>= 5.0)
+    web-console (4.1.0)
+      actionview (>= 6.0.0)
+      activemodel (>= 6.0.0)
       bindex (>= 0.4.0)
-      railties (>= 5.0)
-    websocket-driver (0.6.5)
+      railties (>= 6.0.0)
+    websocket-driver (0.7.3)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.5)
     xpath (3.2.0)
       nokogiri (~> 1.8)
+    zeitwerk (2.4.2)
 
 PLATFORMS
   ruby
 
 DEPENDENCIES
   byebug
-  capybara (~> 2.13)
-  coffee-rails (~> 4.2)
-  jbuilder (~> 2.5)
-  listen (>= 3.0.5, < 3.2)
+  capybara
+  coffee-rails
+  jbuilder
+  listen
+  mini_portile2
```

Comment:
This shouldn't be necessary, see above.

Reply:
Ideally, no, but how'd you create a multi-platform Gemfile.lock? When I generate it, on x86_64, I won't get this dependency. This works fine as a workaround.

Reply:
I managed to do it in the end.